origin.spec

```diff
@@ -103,6 +414,7 @@ Obsoletes:      openshift-clients-redistributable < %{package_refector_version}
 
 %description clients-redistributable
 %{summary}
+%endif
```

Comment:
Well in Fedora, they would just install the client package that's native to Fedora. The assumption here is that a Fedora user wouldn't need the OS X or Windows client.